include/avif/internal.h

```diff
@@ -15,26 +15,44 @@ extern "C" {
 #define AVIF_MIN(a, b) (((a) < (b)) ? (a) : (b))
 #define AVIF_MAX(a, b) (((a) > (b)) ? (a) : (b))
 
+// Used for debugging. Add ! to catch the earliest failure during encoding or decoding.
+#if defined(AVIF_BREAK_ON_ERROR)
+static inline void avifBreakOnError()
+{
+    // Same mechanism as OpenCV's error() function.
+    static volatile int * p = 0;
+    *p = 0; // or replace by a breakpoint
```

Comment:
Nit: This comment should be moved before line 23, because it now describes two lines (23-24). Originally it described only this line when this line was `abort();`.